src/main/java/org/kainos/ea/controller/JobRoleController.java

```diff
@@ -35,4 +34,18 @@ public Response getAllJobRoles() {
             return Response.serverError().build();
         }
     }
+
+    @GET
+    @Path("/capability")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getCapability() {
+
+        try {
+            return Response.ok(jobRoleService.getCapability()).build();
+        } catch (FailedToGetCapabilityException e) {
+            System.err.println(e.getMessage());
+
+            return Response.serverError().build();
+        }
+    }
```

Comment:
you don't need a new endpoint, you can just return the capability as part of the existing get roles endpoint for US001